fem/bilinearform_ext.cpp

```diff
@@ -295,7 +295,14 @@ void PABilinearFormExtension::Assemble()
    const int integratorCount = integrators.Size();
    for (int i = 0; i < integratorCount; ++i)
    {
-      integrators[i]->AssemblePA(*a->FESpace());
+      if (a->FESpace()->GetMesh()->NURBSext && integrators[i]->Patchwise())
```

Comment:
Check for NURBS FE space, instead of just checking for NURBS mesh:
```suggestion
      if (a->FESpace()->GetNURBSext() && integrators[i]->Patchwise())
```

Reply:
It maybe better to dispatch to `AssembleNURBSPA` or `AssemblePA` just based on `integrators[i]->Patchwise()` and trigger an error if patch-wise integration is requested but the space is not NURBS FE space. What do you think?

Reply:
Done.